app/icon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Right but the more "Componenty" approach for the extension would be composition:

```
<Icon>
 <path>…
</Icon>
```

the `switch` case would conflate all the possible icon combinations.